docs/source/dataframe-api.rst

```diff
@@ -432,3 +438,10 @@ Other functions
 .. autofunction:: get_dummies
 .. autofunction:: pivot_table
 .. autofunction:: melt
+
+Demo Data
+~~~~~~~~~
+
+.. currentmodule:: dask.dataframe
+
+.. autofunction::  demo.make_timeseries
```

Comment:
I wouldn't mind deprecating this module.  I've been using `dask.datasets.timeseries` instead lately.

Reply:
Oh, I got the old vs. new modules mixed up. I'll use datasets.